mmv1/products/databasemigrationservice/connectionprofile.yaml

```diff
@@ -336,15 +354,194 @@ properties:
           Output only. If the source is a Cloud SQL database, this field indicates the network architecture it's associated with.
         values:
           - :NETWORK_ARCHITECTURE_OLD_CSQL_PRODUCER
-          - :NETWORK_ARCHITECTURE_NEW_CSQL_PRODUCER -
-            !ruby/object:Api::Type::NestedObject
+          - :NETWORK_ARCHITECTURE_NEW_CSQL_PRODUCER
+      - !ruby/object:Api::Type::NestedObject
```

Comment:
it could be nice to split out the postgres changes into a separate PR since they're straightforward & have automated tests & would be easy to review.

Reply:
https://github.com/GoogleCloudPlatform/magic-modules/pull/9137